src/types/response/spot-margin-trading.ts

```diff
@@ -14,39 +14,48 @@
 
 export interface HFMarginOrder {
   id: string;
+  clientOid: string;
   symbol: string;
-  opType: 'DEAL';
+  opType: string;
   type: 'limit' | 'market';
   side: 'buy' | 'sell';
   price: string;
   size: string;
   funds: string;
-  dealFunds: string;
   dealSize: string;
+  dealFunds: string;
+  remainSize: string;
+  remainFunds: string;
+  cancelledSize: string;
+  cancelledFunds: string;
   fee: string;
   feeCurrency: string;
-  stp: string;
+  stp?: 'DC' | 'CO' | 'CN' | 'CB' | null;
+  stop?: string | null;
+  stopTriggered: boolean;
+  stopPrice: string;
   timeInForce: 'GTC' | 'GTT' | 'IOC' | 'FOK';
   postOnly: boolean;
   hidden: boolean;
   iceberg: boolean;
   visibleSize: string;
   cancelAfter: number;
   channel: string;
-  clientOid: string;
-  remark: string;
-  tags: string;
-  active: boolean;
-  inOrderBook: boolean;
+  remark?: string | null;
+  tags?: string | null;
   cancelExist: boolean;
+  tradeType: string;
+  inOrderBook: boolean;
+  tax: string;
+  active: boolean;
   createdAt: number;
   lastUpdatedAt: number;
-  tradeType: 'MARGIN_TRADE' | 'MARGIN_ISOLATED_TRADE';
 }
 
+// deprecated, not used anymore
```

Comment:
Use jsdoc @deprecated for that, so it shows in IDEs
```suggestion
/** @deprecated not used anymore **/
```